Bottom: ee1de1c781287d8120f10c1374fabab5ebc32a63
Top:    4d04f6d05a3492b996abd2f4910e0efbe324f5f6
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-04-09 08:19:32 -0300

Refresh of revamp-cdtor-api.patch

---

diff --git a/include/gcc-cp-interface.h b/include/gcc-cp-interface.h
index e932a18..3ae4def 100644
--- a/include/gcc-cp-interface.h
+++ b/include/gcc-cp-interface.h
@@ -41,7 +41,7 @@ struct gcc_cp_context;
 
 enum gcc_cp_api_version
 {
-  GCC_CP_FE_VERSION_0 = 0xffffffff-8
+  GCC_CP_FE_VERSION_0 = 0xffffffff-9
 };
 
 /* Qualifiers.  */
